### ISSUE TITLE
Suggestion: Detect if addon is available on AMO

### ISSUE THREAD

**Issue by trlkly** [Noitidart/Chrome-Store-Foxified#39]
Obviously, it might not be the exact same addon. But you could at least let people know if there is an addon with the same or similar name on AMO, and provide a button to go look at that one instead. (But still let them install the Google version if they want.)

As you said in another bug, it's generally better to install the Firefox version of an addon if it exists, so I thought it might be a good idea to facilitate this.

It could also be an alternative to caching signed addons. The most popular addons will probably get an AMO release.

**Comment by Noitidart** [comment]
This would be super cool. I have some ideas on how to detect if a Firefox version exists but none of them are sure fire:

* Search if the id is found on AMO
* Search by name

Maybe just provide a "suggestions" which populates with these search results.

**Comment by trlkly** [comment]
Yeah, that's basically what I was thinking. Since you put out a popup when
you click the button on the page, the search results could show there,
maybe even with the "Add to Firefox" button.

If there's one with the same ID, you could highlight it in the results.

**Comment by revad** [comment]
I think this is quite important. I can see some potential problems for the developer if an addon is also available on AMO and people are -  unknown to them - installing the chrome version.  Stats? Reviews? And for the user: how does it handle updates? 

I have an addon in both  stores which is different - the Fx one does more. And recently I had to update the Fx one with workround for a Firefox bug - so the Chrome one wouldn't work.  It's not a webextension yet because it doesn't work on Android - but it will be.

One easy thing to check is whether the manifest contains applications: gecko and the id.  

I have an addon in the chrome store which exists in AMO but is not public. So an attempt to alert me that someone has loaded it from the Chrome Store would be useful.

**Comment by Noitidart** [comment]
Thanks revad for your vote. I'm currently trying to make all my addons future proof, as they have decided to ban all these non-webext this year. As soon as I do that I'll be free to work on features like this.
